lib/model/parameter/SortBy.dart

```diff
@@ -27,3 +20,18 @@ enum SortOption {
   CREATED,
   EDIT,
 }
+
+extension SortOptionExt on SortOption {
+  String get name {
```

Comment:
@blazern I know that you did not agree, but from #184 there's a sort of consensus about using `Map<MyEnum, String>`. The way it was coded in this file just before your PR ;)

Reply:
Returned back to how it was before